package.json

```diff
@@ -12,19 +12,28 @@
   ],
   "license": "MPL-2.0",
   "scripts": {
+    "ts": "tsc",
+    "ts-watch": "tsc -w",
+    "lint": "standard --verbose",
+    "lint-fix": "standard --fix --verbose",
     "bootstrap": "lerna bootstrap",
```

Comment:
would love to group this 3 together as well. When do we use bootstrap and build?

Reply:
@cezaraugusto

Reply:
we're not using bootstrap anymore so it's more like a leftover, I'm ok grouping other tasks

Reply:
removed